source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -887,6 +887,24 @@ class NiDAQmxDriverApiTests : public Test {
     return status;
   }
 
+  ::grpc::Status perform_bridge_offset_nulling_cal_ex(PerformBridgeOffsetNullingCalExResponse& response)
+  {
+    ::grpc::ClientContext context;
+    PerformBridgeOffsetNullingCalExRequest request;
```

Comment:
This is not passing the task name or any of the other parameters.

Add parameters to this function and set them on the request object:
```
    set_request_session_name(request);
    request.set_channel(channel);
    request.set_skip_unsupported_channels(skip_unsupported_channels);
```

Reply:
BTW, the tests caught this.

```
[ RUN      ] NiDAQmxDriverApiTests.BridgeOffsetNullingCal_Succeeds
unknown file: Failure
C++ exception with description "Task specified is invalid or does not exist.
Status Code: -200088" thrown in the test body.
[  FAILED  ] NiDAQmxDriverApiTests.BridgeOffsetNullingCal_Succeeds (2 ms)
[ RUN      ] NiDAQmxDriverApiTests.ThrmcplLeadOffsetNullingCal_Succeeds
unknown file: Failure
C++ exception with description "Task specified is invalid or does not exist.
Status Code: -200088" thrown in the test body.
[  FAILED  ] NiDAQmxDriverApiTests.ThrmcplLeadOffsetNullingCal_Succeeds (3 ms)
```

Reply:
Thanks for pointing this out, I have added parameters to these functions and set them on the request object for each of the functions. However I am unable to test it for now because I'm facing build issues, I am just eyeballing it and will test it once I am able to.